scripts/rgbd_monitor

```diff
@@ -16,9 +16,10 @@ class RGBDMonitor(object):
         """ Constructor
         :param timeout: if no messages have been received for <timeout> seconds, the rgbd to ros converter is killed
         """
+        self._ns = rospy.get_namespace()
         self._last_stamp = None
         self._timeout = rospy.Duration(timeout)
-        self._image_sub = rospy.Subscriber('/amigo/top_kinect/rgb/image', Image, self._image_callback)
+        self._image_sub = rospy.Subscriber('rgb/image', Image, self._image_callback)
```

Comment:
Don't you need the namespace here?

Reply:
No, because this is a relative topic name. ROS takes care of it in a correct way.